jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/delegate/AbstractFieldDelegate.java

```diff
@@ -161,6 +161,14 @@ public void executeValidators() throws ValidationException {
         // the component can be obsolete in validation time.
         T value = modelValue != null ? modelValue : getComponentValue();
 
+        if (value == null && component.getElement().getProperty("_hasInputValue", false)) {
+            setInvalidInternal(true);
+            String validationMessage = messages.formatMessage(
```

Comment:
No reason to use `messages.formatMessage` without passing parameters. Is it hot in the office?)